Binary files MonoGame-3.4.orig/IDE/MonoDevelop/bin/Release/MonoDevelop.MonoGame.Core.dll and MonoGame-3.4/IDE/MonoDevelop/bin/Release/MonoDevelop.MonoGame.Core.dll differ
Binary files MonoGame-3.4.orig/IDE/MonoDevelop/bin/Release/MonoDevelop.MonoGame.Core.dll.mdb and MonoGame-3.4/IDE/MonoDevelop/bin/Release/MonoDevelop.MonoGame.Core.dll.mdb differ
Binary files MonoGame-3.4.orig/IDE/MonoDevelop/bin/Release/MonoDevelop.MonoGame.dll and MonoGame-3.4/IDE/MonoDevelop/bin/Release/MonoDevelop.MonoGame.dll differ
Binary files MonoGame-3.4.orig/IDE/MonoDevelop/bin/Release/MonoDevelop.MonoGame.dll.mdb and MonoGame-3.4/IDE/MonoDevelop/bin/Release/MonoDevelop.MonoGame.dll.mdb differ
Binary files MonoGame-3.4.orig/IDE/MonoDevelop/MonoDevelop.MonoGame/obj/Release/MonoDevelop.MonoGame.dll and MonoGame-3.4/IDE/MonoDevelop/MonoDevelop.MonoGame/obj/Release/MonoDevelop.MonoGame.dll differ
Binary files MonoGame-3.4.orig/IDE/MonoDevelop/MonoDevelop.MonoGame/obj/Release/MonoDevelop.MonoGame.dll.mdb and MonoGame-3.4/IDE/MonoDevelop/MonoDevelop.MonoGame/obj/Release/MonoDevelop.MonoGame.dll.mdb differ
Binary files MonoGame-3.4.orig/IDE/MonoDevelop/MonoDevelop.MonoGame.Core/obj/Release/MonoDevelop.MonoGame.Core.dll and MonoGame-3.4/IDE/MonoDevelop/MonoDevelop.MonoGame.Core/obj/Release/MonoDevelop.MonoGame.Core.dll differ
Binary files MonoGame-3.4.orig/IDE/MonoDevelop/MonoDevelop.MonoGame.Core/obj/Release/MonoDevelop.MonoGame.Core.dll.mdb and MonoGame-3.4/IDE/MonoDevelop/MonoDevelop.MonoGame.Core/obj/Release/MonoDevelop.MonoGame.Core.dll.mdb differ
diff -ur MonoGame-3.4.orig/MonoGame.Framework.Content.Pipeline/Graphics/Font/SharpFontImporter.cs MonoGame-3.4/MonoGame.Framework.Content.Pipeline/Graphics/Font/SharpFontImporter.cs
--- MonoGame-3.4.orig/MonoGame.Framework.Content.Pipeline/Graphics/Font/SharpFontImporter.cs	2016-01-20 18:29:19.692189781 -0800
+++ MonoGame-3.4/MonoGame.Framework.Content.Pipeline/Graphics/Font/SharpFontImporter.cs	2016-01-20 18:59:13.216653096 -0800
@@ -52,10 +52,10 @@
 					Glyphs = glyphList;
 
 					// Store the font height.
-					LineSpacing = face.Size.Metrics.Height >> 6;
+					LineSpacing = (int)face.Size.Metrics.Height >> 6;
 
 					// The height used to calculate the Y offset for each character.
-					YOffsetMin = -face.Size.Metrics.Ascender >> 6;
+					YOffsetMin = -(int)face.Size.Metrics.Ascender >> 6;
 				}
 			} finally {
 				if (face != null)
@@ -123,8 +123,8 @@
 				glyphBitmap = bitmap.Clone(new System.Drawing.Rectangle(0, 0, face.Glyph.Bitmap.Width, face.Glyph.Bitmap.Rows), PixelFormat.Format32bppArgb);
 			else 
 			{
-				var gHA = face.Glyph.Metrics.HorizontalAdvance >> 6;
-				var gVA = face.Size.Metrics.Height >> 6;
+				var gHA = (int)face.Glyph.Metrics.HorizontalAdvance >> 6;
+				var gVA = (int)face.Size.Metrics.Height >> 6;
 
 				gHA = gHA > 0 ? gHA : gVA;
 				gVA = gVA > 0 ? gVA : gHA;
@@ -136,16 +136,16 @@
 
 			// not sure about this at all
 			var abc = new ABCFloat ();
-			abc.A = face.Glyph.Metrics.HorizontalBearingX >> 6;
-			abc.B = face.Glyph.Metrics.Width >> 6;
-			abc.C = (face.Glyph.Metrics.HorizontalAdvance >> 6) - (abc.A + abc.B);
+			abc.A = (int)face.Glyph.Metrics.HorizontalBearingX >> 6;
+			abc.B = (int)face.Glyph.Metrics.Width >> 6;
+			abc.C = ((int)face.Glyph.Metrics.HorizontalAdvance >> 6) - (abc.A + abc.B);
 
 			// Construct the output Glyph object.
 			return new Glyph(character, glyphBitmap)
 			{
-				XOffset = -(face.Glyph.Advance.X >> 6),
-				XAdvance = face.Glyph.Metrics.HorizontalAdvance >> 6,
-                YOffset = -(face.Glyph.Metrics.HorizontalBearingY >> 6),
+				XOffset = -((int)face.Glyph.Advance.X >> 6),
+				XAdvance = (int)face.Glyph.Metrics.HorizontalAdvance >> 6,
+                YOffset = -((int)face.Glyph.Metrics.HorizontalBearingY >> 6),
 				CharacterWidths = abc
 			};
 		}
